src/pages/index.js

```diff
@@ -60,28 +107,122 @@ document.addEventListener("DOMContentLoaded", function () {
     addFormValidator.resetValidation();
   });
 
-  newCardList.renderItems();
+  constants.avatar.addEventListener("click", () => {
+    avatarPopup.open();
+    addFormValidator.resetValidation();
+  });
+
+  constants.avatar.addEventListener("mousemove", () => {
+    constants.avatarButton.classList.replace(
+      "profile__avatar-button_disabled",
+      "profile__avatar-button"
+    );
+  });
+});
+
+constants.avatar.addEventListener("mouseout", () => {
+  constants.avatarButton.classList.replace(
+    "profile__avatar-button",
+    "profile__avatar-button_disabled"
+  );
 });
 
 /* -------------------------------------------------------------------------- */
 /*                                  Functions                                 */
 /* -------------------------------------------------------------------------- */
 function createCard(cardData) {
-  const card = new Card(cardData, "#card-template", handleImageClick);
+  card = new Card(
```

Comment:
`let card;` is not needed here because you shouldn't reassign the variable for each card you create. It can be misleading


So, here should be `const card =` which you had previously